x-pack/index.js

```diff
@@ -5,9 +5,8 @@
  */
 
 import { xpackMain } from './legacy/plugins/xpack_main';
-import { security } from './legacy/plugins/security';
```

Comment:
__nit:__ would you mind removing two `x-pack/legacy/plugins/security` entries from the CODEOWNERS file too?

Reply:
and... It's gone! https://github.com/elastic/kibana/pull/75648/commits/2177028aa1a67c8316fa750382b27603ab594c0d